### ISSUE TITLE
Keyboard navigation doesn't work in IE9

### ISSUE THREAD

**Issue by dandv** [hakimel/reveal.js#202]
I'm not sure, but I think this used to work in a previous version; now, arrow keys, Home/End/PgUp/PgDn don't do anything in IE9. To reproduce, just open the `index.html` as of the last master commit.
